staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder/builder_test.go

```diff
@@ -520,7 +520,7 @@ func TestCRDRouteParameterBuilder(t *testing.T) {
 				hasNamespaceParam := false
 				hasNameParam := false
 				for _, param := range path.Parameters {
-					if param.In == "path" && param.Name == "namespace" {
+					if strings.HasPrefix(param.Ref.String(), "#/parameters/namespace-") {
```

Comment:
I'm surprised this changed... what got deduplicated that changed the namespace parameter name?

Reply:
Oh, I see… we always generate a ref named with a hash of the content

Reply:
Compare https://github.com/kubernetes/kube-openapi/blob/master/pkg/builder/parameters.go#L64. Some parameters have the kind name in their description and/or the schema is kind specific. The name is like that. Namespace is generic and hence we moved it into the global parameters block. This change here fixes the test looking for that parameter.

Reply:
The hash is for best-effort uniqueness of shared parameters. We have the code to rename parameters on aggregation if they differ. But depending on renaming (when not having the hash) would make the spec to change when APIServices come and go. With the hash this won‘t happen in practice.